docs/reference/plugins.rst

```diff
@@ -16,6 +16,20 @@
     limitations under the License.
 
 
+Plugin Gallery
```

Comment:
as previously stated i'm strongly against aggregating in a static document dynamically created plugins. i can list here as well my reasons regarding this if it's required.

Reply:
thanks viktor - we will close this for now.